src/app-logic/l10n.js

```diff
@@ -10,28 +10,41 @@ import {
   PSEUDO_STRATEGIES_DIRECTION,
 } from 'firefox-profiler/utils/l10n-pseudo';
 
-// This contains the locales we support in the production. Don't forget to update
-// the array RTL_LOCALES when adding a RTL locale, if necessary.
-// AVAILABLE_STAGING_LOCALES is replaced, using webpack's DefinePlugin, by all
-// available locales, when running with L10N=1, using the "l10n" versions of the
-// yarn scripts. Especially this is done when building and deploying the l10n
-// branch in netlify.
-export const AVAILABLE_LOCALES: Array<string> = AVAILABLE_STAGING_LOCALES || [
-  'de',
-  'el',
-  'en-GB',
-  'en-US',
-  'es-CL',
-  'fr',
-  'ia',
-  'it',
-  'nl',
-  'pt-BR',
-  'sv-SE',
-  'uk',
-  'zh-CN',
-  'zh-TW',
-];
+// This object contains the locales that are at least 90% complete and
+// that we enable by default. Don't forget to update the array RTL_LOCALES when
+// adding a RTL locale, if necessary.
+// The localized names are copied from the curated list that can be found at
+// https://mozilla-l10n.github.io/firefox-languages/.
+// We use the language names in their own language.
+// Also note that the order specified here is the order they'll be displayed in the
+// language switcher, so it's important to keep the alphabetical order.
```

Comment:
Note: I checked locally using L10N=1 and at least on my system, this looks also properly ordered. It would be good to double check on other systems, we may need to sort it in `webpack.config.js`.

Reply:
It looks sorted on my machine as well on macOS. I think it was fine even if the non-enabled locales were out of order since that's mostly for debugging and localization and not for production.